src/Storages/MergeTree/BackgroundProcessingPool.cpp

```diff
@@ -51,12 +45,12 @@ BackgroundProcessingPool::BackgroundProcessingPool(int size_,
     , thread_name(thread_name_)
     , settings(pool_settings)
 {
-    logger = &Poco::Logger::get(log_name);
-    LOG_INFO(logger, "Create {} with {} threads", log_name, size);
+    logger = &Logger::get(log_name);
+    LOG_INFO(logger, "Create " << log_name << " with " << size << " threads");
```

Comment:
There were some minor updates in the logger so this changes are not needed #11137 #11272